cinema_service/settings.py

```diff
@@ -20,14 +22,14 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = os.getenv("SECRET_KEY")
```

Comment:
The 'SECRET_KEY' should have a default value for the cases when the environment variable is not defined. This is useful for local development and testing.